lib/ansible/modules/cloud/vmware/vmware_cluster.py

```diff
@@ -360,11 +136,11 @@ def state_create_cluster(self):
             self.module.fail_json(msg="Failed to create cluster"
                                       " due to generic exception %s" % to_native(generic_exc))
 
-    def state_destroy_cluster(self):
+    def destroy_cluster(self):
         """
         Destroy cluster
         """
-        changed, result = False, None
+        changed, result = True, None
```

Comment:
```suggestion
        changed, result = False, None
```
`changed` should be False until an actual change action sets it to True.